--- gdb.spec.orig	2015-12-06 04:10:30.000000000 +0000
+++ gdb.spec	2016-01-20 14:49:12.745843903 +0000
@@ -16,7 +16,10 @@
 }
 
 Summary: A GNU source-level debugger for C, C++, Fortran, Go and other languages
-Name: %{?scl_prefix}gdb
+Name: %{?scl_prefix}scylla-gdb
+%define orig_name gdb
+Requires:	scylla-env
+%define _prefix /opt/scylladb
 
 # Freeze it when GDB gets branched
 %global snapsrc    20150706
@@ -572,12 +575,8 @@
 BuildRequires: rpm-devel%{buildisa}
 BuildRequires: zlib-devel%{buildisa} libselinux-devel%{buildisa}
 %if 0%{!?_without_python:1}
-%if 0%{?rhel:1} && 0%{?rhel} <= 7
-BuildRequires: python-devel%{buildisa}
-%else
-%global __python %{__python3}
-BuildRequires: python3-devel%{buildisa}
-%endif
+BuildRequires: python34-devel%{?_isa}
+%global __python /usr/bin/python3.4
 %if 0%{?rhel:1} && 0%{?rhel} <= 7
 # Temporarily before python files get moved to libstdc++.rpm
 # libstdc++%{bits_other} is not present in Koji, the .spec script generating
