--- isl.spec.orig	2016-01-20 14:41:16.891802146 +0000
+++ isl.spec	2016-01-20 14:43:13.838336396 +0000
@@ -1,5 +1,5 @@
 Summary: Integer point manipulation library
-Name: isl
+Name: scylla-isl
 Version: 0.14
 License: MIT
 Group: System Environment/Libraries
@@ -17,8 +17,10 @@
 
 BuildRequires: gmp-devel
 BuildRequires: pkgconfig
+Requires: scylla-env
 
 Source0: http://isl.gforge.inria.fr/isl-%{version}.tar.xz
+%define _prefix /opt/scylladb
 
 %description
 isl is a library for manipulating sets and relations of integer points
@@ -32,7 +34,8 @@
 
 %package devel
 Summary: Development for building integer point manipulation library
-Requires: isl%{?_isa} == %{version}-%{release}
+Requires: scylla-env
+Requires: scylla-isl%{?_isa} == %{version}-%{release}
 Requires: gmp-devel%{?_isa}
 Group: Development/Libraries
 
@@ -47,7 +50,7 @@
 graphs), dependence analysis and bounds on piecewise step-polynomials.
 
 %prep
-%setup -q
+%setup -q -n isl-%{version}
 
 %build
 %configure
