diff -Nur debian/changelog ../thrift-0.10.0.new/debian/changelog
--- debian/changelog	2016-12-19 20:05:45.000000000 +0000
+++ ../thrift-0.10.0.new/debian/changelog	2017-06-20 19:14:33.902186804 +0000
@@ -1,3 +1,8 @@
+thrift (0.10.0-1) stable; urgency=low
+  * update to 0.10.0
+
+ -- Takuya ASADA <syuu@scylladb.com>  Mon, 19 Jun 2017 23:08:43 +0000
+
 thrift (0.10.0) stable; urgency=low
 
   * update to 0.10.0
diff -Nur debian/control ../thrift-0.10.0.new/debian/control
--- debian/control	2016-12-21 03:04:19.000000000 +0000
+++ ../thrift-0.10.0.new/debian/control	2017-06-20 19:14:33.902186804 +0000
@@ -1,14 +1,10 @@
 Source: thrift
 Section: devel
 Priority: extra
-Build-Depends: debhelper (>= 9), build-essential, mono-mcs, python-dev, ant,
-    mono-devel,  libmono-system-web4.0-cil, erlang-base, ruby-dev | ruby1.9.1-dev, ruby-bundler ,autoconf, automake,
+Build-Depends: debhelper (>= 9), build-essential, autoconf, automake,
     pkg-config, libtool, bison, flex, libboost-dev | libboost1.53-dev,
-    python-all, python-setuptools, python-all-dev, python-all-dbg,
-    python3-all, python3-setuptools, python3-all-dev, python3-all-dbg,
-    openjdk-7-jdk | openjdk-8-jdk | default-jdk,
-    libboost-test-dev | libboost-test1.53-dev, libevent-dev, libssl-dev, perl (>= 5.8.0-7),
-    php5 | php7.0, php5-dev | php7.0-dev, libglib2.0-dev, qtchooser, qtbase5-dev-tools
+    libboost-test-dev | libboost-test1.53-dev, libevent-dev, libssl-dev, 
+    libglib2.0-dev, qtchooser, qtbase5-dev-tools
 Maintainer: Thrift Developer's <dev@thrift.apache.org>
 Homepage: http://thrift.apache.org/
 Vcs-Git: https://git-wip-us.apache.org/repos/asf/thrift.git
@@ -17,144 +13,6 @@
 X-Python-Version: >= 2.6
 X-Python3-Version: >= 3.3
 
-Package: thrift-compiler
-Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}
-Description: Compiler for Thrift definition files
- Thrift is a software framework for scalable cross-language services
- development. It combines a software stack with a code generation engine to
- build services that work efficiently and seamlessly.
- .
- This package contains the Thrift compiler that is used for translating
- from .thrift files (containing the definitions) to the language binding
- for the supported languages.
-
-Package: python-thrift
-Architecture: any
-Section: python
-Depends: ${python:Depends}, ${shlibs:Depends}, ${misc:Depends}, python-six
-Recommends: python-twisted-web, python-backports.ssl-match-hostname, python-ipaddress
-Provides: ${python:Provides}
-Description: Python bindings for Thrift (Python 2)
- Thrift is a software framework for scalable cross-language services
- development. It combines a software stack with a code generation engine to
- build services that work efficiently and seamlessly.
- .
- This package contains the Python bindings for Thrift. You will need the thrift
- tool (in the thrift-compiler package) to compile your definition to Python
- classes, and then the modules in this package will allow you to use those
- classes in your programs.
- .
- This package installs the library for Python 2.
-
-Package: python-thrift-dbg
-Architecture: any
-Section: debug
-Depends: ${shlibs:Depends}, ${misc:Depends}, python-thrift (= ${binary:Version}), python-all-dbg
-Provides: ${python:Provides}
-Description: Python bindings for Thrift (debug version)
- Thrift is a software framework for scalable cross-language services
- development. It combines a software stack with a code generation engine to
- build services that work efficiently and seamlessly.
- .
- This package contains the Python bindings for Thrift with debugging symbols.
- You will need the thrift tool (in the thrift-compiler package) to compile your
- definition to Python classes, and then the modules in this package will allow
- you to use those classes in your programs.
-
-Package: python3-thrift
-Architecture: any
-Section: python
-Depends: ${python3:Depends}, ${shlibs:Depends}, ${misc:Depends}, python3-six
-Recommends: python3-twisted-web
-Provides: ${python:Provides}
-Description: Python bindings for Thrift (Python 3)
- Thrift is a software framework for scalable cross-language services
- development. It combines a software stack with a code generation engine to
- build services that work efficiently and seamlessly.
- .
- This package contains the Python bindings for Thrift. You will need the thrift
- tool (in the thrift-compiler package) to compile your definition to Python
- classes, and then the modules in this package will allow you to use those
- classes in your programs.
- .
- This package installs the library for Python 3.
-
-Package: python3-thrift-dbg
-Architecture: any
-Section: debug
-Depends: ${shlibs:Depends}, ${misc:Depends}, python3-thrift (= ${binary:Version}), python3-all-dbg
-Provides: ${python:Provides}
-Description: Python bindings for Thrift (debug version)
- Thrift is a software framework for scalable cross-language services
- development. It combines a software stack with a code generation engine to
- build services that work efficiently and seamlessly.
- .
- This package contains the Python bindings for Thrift with debugging symbols.
- You will need the thrift tool (in the thrift-compiler package) to compile your
- definition to Python classes, and then the modules in this package will allow
- you to use those classes in your programs.
-
-Package: ruby-thrift
-Architecture: all
-Section: libs
-Depends: ruby | ruby-interpreter, ${shlibs:Depends}, ${misc:Depends}
-Provides: libthrift-ruby
-Replaces: libthrift-ruby
-Breaks: libthrift-ruby
-Description: Ruby bindings for Thrift
- Thrift is a software framework for scalable cross-language services
- development. It combines a software stack with a code generation engine to
- build services that work efficiently and seamlessly.
- .
- This package contains the Ruby bindings for Thrift. You will need the thrift
- tool (in the thrift-compiler package) to compile your definition to Ruby
- classes, and then the modules in this package will allow you to use those
- classes in your programs.
-
-Package: libthrift-java
-Architecture: all
-Section: java
-Depends: ${misc:Depends}
-Description: Java bindings for Thrift
- Thrift is a software framework for scalable cross-language services
- development. It combines a software stack with a code generation engine to
- build services that work efficiently and seamlessly.
- .
- This package contains the Java bindings for Thrift. You will need the thrift
- tool (in the thrift-compiler package) to compile your definition to Java
- classes, and then the modules in this package will allow you to use those
- classes in your programs.
-
-Package: libthrift-cil
-Architecture: all
-Section: cli-mono
-Depends: cli-common, libmono-corlib4.0-cil (>= 2.10) | libmono-corlib4.5-cil (>=3.2), libmono-system4.0-cil (>= 2.10),
-    libmono-system-web4.0-cil (>= 2.10), ${misc:Depends}
-Description: CLI bindings for Thrift
- Thrift is a software framework for scalable cross-language services
- development. It combines a software stack with a code generation engine to
- build services that work efficiently and seamlessly.
- .
- This package contains the CLI bindings for Thrift. You will need the thrift
- tool (in the thrift-compiler package) to compile your definition to C#
- classes, and then the modules in this package will allow you to use those
- classes in your programs.
-
-Package: libthrift-perl
-Architecture: all
-Section: perl
-Depends: perl (>= 5.8.0-7), ${misc:Depends}
-Description: Perl bindings for Thrift
- Thrift is a software framework for scalable cross-language services
- development. It combines a software stack with a code generation engine to
- build services that work efficiently and seamlessly.
- .
- This package contains the Perl bindings for Thrift. You will need the thrift
- tool (in the thrift-compiler package) to compile your definition to Perl
- classes, and then the modules in this package will allow you to use those
- classes in your programs.
-
 Package: libthrift0
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}
@@ -177,15 +35,3 @@
  .
  This package contains the development libraries required for writing C++
  applications using Thrift.
-
-Package: php5-thrift
-Architecture: any
-Section: php
-Depends: ${php:Depends}, ${shlibs:Depends}, ${misc:Depends}
-Provides: ${php:Provides}
-Description: PHP bindings for Thrift
- Thrift is a software framework for scalable cross-language services
- development. It combines a software stack with a code generation engine to
- build services that work efficiently and seamlessly.
- .
- This package contains the PHP bindings for Thrift.
diff -Nur debian/libthrift0.install ../thrift-0.10.0.new/debian/libthrift0.install
--- debian/libthrift0.install	2016-04-03 17:19:43.000000000 +0000
+++ ../thrift-0.10.0.new/debian/libthrift0.install	2017-06-20 19:22:46.321957388 +0000
@@ -1,5 +1,4 @@
 usr/lib/libthrift.so*
 usr/lib/libthrift-*.so*
 usr/lib/libthriftnb*.so*
-usr/lib/libthriftqt*.so*
 usr/lib/libthriftz*.so*
diff -Nur debian/rules ../thrift-0.10.0.new/debian/rules
--- debian/rules	2016-12-19 20:05:45.000000000 +0000
+++ ../thrift-0.10.0.new/debian/rules	2017-06-20 19:22:48.285956469 +0000
@@ -51,18 +51,6 @@
 	# Compile C (glib) library
 	$(MAKE) -C $(CURDIR)/lib/c_glib
 
-	# Python library
-	cd $(CURDIR)/lib/py && \
-    for py in $(PYVERS); do  \
-        $$py setup.py build; \
-        $$py-dbg setup.py build; \
-    done
-
-	# PHP
-	cd $(CURDIR)/lib/php/src/ext/thrift_protocol && \
-		phpize && \
-		./configure && $(MAKE)
-
 	touch $@
 
 build-indep: build-indep-stamp
@@ -71,19 +59,6 @@
 	# Add here commands to compile the indep part of the package.
 	#$(MAKE) doc
 
-	# Java
-	cd $(CURDIR)/lib/java && \
-		ant
-
-	# C#
-	$(MAKE) -C $(CURDIR)/lib/csharp
-
-	# Ruby
-	$(MAKE) -C $(CURDIR)/lib/rb
-
-	# Perl
-	$(MAKE) -C $(CURDIR)/lib/perl INSTALLDIRS=vendor
-
 	touch $@
 
 clean:
@@ -96,8 +71,6 @@
 	# Add here commands to clean up after the build process.
 	-$(MAKE) clean
 
-	$(CURDIR)/cleanup.sh
-
 	dh_clean
 
 install: install-indep install-arch
@@ -111,29 +84,6 @@
 	# debian/<package>-doc.
 	#INSTALLDOC#
 
-	# Java
-	mkdir -p $(CURDIR)/debian/libthrift-java/usr/share/java/ && \
-	cp $(CURDIR)/lib/java/build/libthrift*.jar \
-		$(CURDIR)/debian/libthrift-java/usr/share/java/
-
-	# Ruby
-	mkdir -p $(CURDIR)/debian/ruby-thrift/usr/lib/ruby/1.9.1 && \
-	cp $(CURDIR)/lib/rb/lib/thrift.rb \
-		$(CURDIR)/debian/ruby-thrift/usr/lib/ruby/1.9.1
-	cp -r $(CURDIR)/lib/rb/lib/thrift \
-		$(CURDIR)/debian/ruby-thrift/usr/lib/ruby/1.9.1
-
-	# C#
-	mkdir -p $(CURDIR)/debian/libthrift-cil/usr/lib/cli/thrift/ && \
-	cp $(CURDIR)/lib/csharp/Thrift.dll \
-		$(CURDIR)/debian/libthrift-cil/usr/lib/cli/thrift/Thrift.dll
-
-	# Perl
-	$(MAKE) -C $(CURDIR)/lib/perl install DESTDIR=$(CURDIR)/debian/libthrift-perl
-	mv $(CURDIR)/debian/libthrift-perl/usr/local/lib/perl5 $(CURDIR)/debian/libthrift-perl/usr/lib
-	rmdir $(CURDIR)/debian/libthrift-perl/usr/local/lib
-	rmdir $(CURDIR)/debian/libthrift-perl/usr/local
-
 	dh_install -i
 
 install-arch:
@@ -146,40 +96,6 @@
 	# debian/tmp.
 	#$(MAKE) DESTDIR=$(CURDIR)/debian/thrift install
 
-	# Compiler
-	mkdir -p $(CURDIR)/debian/thrift-compiler/usr/bin && \
-	cp $(CURDIR)/compiler/cpp/thrift \
-		$(CURDIR)/debian/thrift-compiler/usr/bin/thrift && \
-	rmdir $(CURDIR)/debian/thrift-compiler/usr/sbin
-
-	# Python
-	cd $(CURDIR)/lib/py && \
-	python2 setup.py install --install-layout=deb --no-compile --root=$(CURDIR)/debian/python-thrift && \
-	python2-dbg setup.py install --install-layout=deb --no-compile --root=$(CURDIR)/debian/python-thrift-dbg && \
-	python3 setup.py install --install-layout=deb --no-compile --root=$(CURDIR)/debian/python3-thrift && \
-	python3-dbg setup.py install --install-layout=deb --no-compile --root=$(CURDIR)/debian/python3-thrift-dbg
-
-	find $(CURDIR)/debian/python-thrift -name "*.py[co]" -print0 | xargs -0 rm -f
-	find $(CURDIR)/debian/python-thrift -name "__pycache__" -print0 | xargs -0 rm -fr
-	find $(CURDIR)/debian/python-thrift-dbg -name "__pycache__" -print0 | xargs -0 rm -fr
-	find $(CURDIR)/debian/python-thrift-dbg -name "*.py[co]" -print0 | xargs -0 rm -f
-	find $(CURDIR)/debian/python-thrift-dbg -name "*.py" -print0 | xargs -0 rm -f
-	find $(CURDIR)/debian/python-thrift-dbg -name "*.egg-info" -print0 | xargs -0 rm -rf
-	find $(CURDIR)/debian/python-thrift-dbg -depth -type d -empty -exec rmdir {} \;
-
-	find $(CURDIR)/debian/python3-thrift -name "*.py[co]" -print0 | xargs -0 rm -f
-	find $(CURDIR)/debian/python3-thrift -name "__pycache__" -print0 | xargs -0 rm -fr
-	find $(CURDIR)/debian/python3-thrift-dbg -name "__pycache__" -print0 | xargs -0 rm -fr
-	find $(CURDIR)/debian/python3-thrift-dbg -name "*.py[co]" -print0 | xargs -0 rm -f
-	find $(CURDIR)/debian/python3-thrift-dbg -name "*.py" -print0 | xargs -0 rm -f
-	find $(CURDIR)/debian/python3-thrift-dbg -name "*.egg-info" -print0 | xargs -0 rm -rf
-	find $(CURDIR)/debian/python3-thrift-dbg -depth -type d -empty -exec rmdir {} \;
-
-	# PHP
-	mkdir -p $(CURDIR)/debian/php5-thrift
-	cd $(CURDIR)/lib/php && \
-		$(MAKE) DESTDIR=$(CURDIR)/debian/php5-thrift install
-
 	# C++ and C (glib)
 	mkdir -p $(CURDIR)/debian/tmp; \
 	cd $(CURDIR)/lib/cpp && \
@@ -201,9 +117,8 @@
 	dh_installexamples
 	dh_installman
 	dh_link
-	dh_strip -ppython-thrift --dbg-package=python-thrift-dbg
-	dh_strip -ppython3-thrift --dbg-package=python3-thrift-dbg
 	dh_strip -pthrift-compiler -plibthrift0
+	dh_strip
 	dh_compress
 	dh_fixperms
 	dh_makeshlibs
@@ -219,7 +134,6 @@
 
 # Build architecture dependent packages using the common target.
 binary-arch: build-arch install-arch
-	echo "php:Depends=phpapi-$(shell php-config5 --phpapi)" > $(CURDIR)/debian/substvars
 	$(MAKE) -f debian/rules DH_OPTIONS=-s binary-common
 
 binary: binary-arch binary-indep
